k8s/predict/prod-airqo-predict-job.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: airqo-predict-job
   namespace: production
 spec:
-  successfulJobsHistoryLimit: 1
+  successfulJobsHistoryLimit: 2
```

Comment:
I guess some brief intro about these jobHistoryLimit configs. Care to that here using comments? :) @123MwanjeMike